README.md

```diff
@@ -1,25 +1,175 @@
-Antivol | Antivol pour manette PS4 | [doc](3dDesigns/anti-theft-ps4/README.md)
-Chassis pour voiture V4 | Picots pour adaptation de la voiture au chassis donkeycar | [doc](3dDesigns/car-chassis/README.md)
-Camera cap holder | A camera cap holder on the chassis for avoid to lose ours camera caps | [doc](3dDesigns/camera-cover/README.md)
+First you should create your own recipe book with your wanted recipes. Indeed, ansible have a configuration file where we specifie what options ( recipe ) we want, so you have to modifie the [donkeycar.yml](ansible/donkeycar.yml) file and comments recipe who you don't want
 
+```yml
+ roles:
+    - role: raspi-config #rasperry pi configuration role
+      become: yes # This role need root privileges
+    - role: car-config # car configurations (ntp, hostname, ds4drv...)
+      become: yes # This role need root privileges
+    - role: shutdown-btn # shutdown button service
+      become: yes # This role need root privileges
+    - role: ps4-controller # install and configure ds4drv
+      become: no # This role no need root privileges
+    - role: openCV # Install openCV (optional)
+      become: yes # This role need root privileges
+    - role: donkeycar # Donkeycar installation 
+      become: no # This role doesn't need root privileges
+    - role: mycar # create car with configurations
+      become: no # This role no need root privileges
+    - role: IR-lap-timer # install dependencies for take charge IR counter
+      become: yes # This role need root privileges
+```
+> 📒 [ansible/donkeycar.yml](ansible/donkeycar.yml) preview
+
+* `raspi-config` : defaults installations like wifi credentials, ssh ...
+* `car-config` : car configurations (ntp, hostname, ds4drv...)
+* `shutdown-btn` : add scripts for the shutdown button ( see [documentation](doc/boutton/Boutton_poussoir.md) and [shematic](doc/schema/schema-electronique.png) )
+* `ps4-controller` : installation of ds4drv and configuration for ps4 bluetooth controllers
+* `openCV` : install openCV according to donkeycar documentation
+* `donkeycar` : install [donkeycar project](https://github.com/autorope/donkeycar) and his requirements 
+* `mycar` : create mycar folder with its files
+* `IR-lap-timer` : Infrared installation and configuration
+
+**Don't comment obligatory recipes (raspi-config, car-config, donkeycar, mycar ), this action can avoid the proper functioning of installations or configurations**
+
+## ⚙️ Modifie default car configurations
+
+***
+
+- modifie configuration for single car
+
+  Before launch your recipe, you should modifie config var with your configurations like `wpa_suplicant`, donkeycar sterring and throttle configuration
+  > 📒 [ansible/group_vars/all](ansible/group_vars/all)
+
+- modifie configuration for multiple cars with the same image
+
+  You can choice configuration for multiples cars. For example we made this in our repo for our 4 cars ( see [ansible/config](ansible/config)).
```

Comment:
You can configure the cars independently, in our case we have 4 configurations (see ...